### DISCUSSION TITLE
Similar package in Python

### DISCUSSION THREAD

**Discussion by MaximilienGaudette** [rstudio/pointblank#403] [General]
Hello, is there a similar package in Python? :)

**Comment by rich-iannone** [comment]
There are quite a few! Here is a listing of the more notable ones:

- https://github.com/pandera-dev/pandera
- https://github.com/great-expectations/great_expectations
- https://github.com/sodadata/soda-core
- https://github.com/re-data/re-data
- https://github.com/cleanlab/cleanlab
- https://github.com/ydataai/pandas-profiling

**Reply by MaximilienGaudette** [reply]
Thank you!!!!

**Comment by sfirke** [comment]
In 2025 this exact package has just been brought to Python! https://github.com/posit-dev/pointblank
